filebeat/module/haproxy/_meta/fields.yml

```diff
@@ -50,17 +50,20 @@
 
         - name: error_message
           description: Error message logged by HAProxy in case of error.
+          # Text needed?
+          # Should it map to ECS?
           type: text
 
         - name: source
-          type: text
+          # What kind of source is this?
+          type: keyword
```

Comment:
This is the syslog source. Akin to the `event.origin` idea someone brought up (which in turn is akin to Filebeat 6.x `source`).

All events have proper `source` and `destination` IP and port already set.

Reply:
Skipping for now?

I wonder if I should leave some comments in the fields.yml like `Future ECS` so we find these easily again?

Reply:
skipping for now